src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    completed,
+  } = todo;
+
+  return (
+    <div
+      className={classNames('todo', {
+        completed,
+      })}
+      key={id}
+    >
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+          checked
```

Comment:
The 'checked' attribute is always set to true, which makes the checkbox always checked. Add a value to the 'checked' attribute based on the 'completed' value from the todo object.